plugins/slack_webhooks/test/decorators/stage_decorator_test.rb

```diff
@@ -27,4 +27,32 @@
       stage.slack_webhooks.size.must_equal 0
     end
   end
+
+  describe "#slack_buddy_channels" do
+    it "returns channels when for_buddy is true" do
+      stage.slack_webhooks = [SlackWebhook.new(channel: "test", webhook_url: "http://slack.com/abcdef",
```

Comment:
Getting complaints from rubocop when I move the next line to 2-space indent instead of aligning with the hash key in the previous line:
```
plugins/slack_webhooks/test/decorators/stage_decorator_test.rb:34:9: C: Align the elements of a hash literal if they span more than one line.
        stage_id: stage.id, for_buddy: true)]
        ^^^^^^^^^^^^^^^^^^
```